src/MetricsUploader/SymbolLoadingMetricTest.cpp

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock-spec-builders.h>
+#include <gtest/gtest.h>
+
+#include <chrono>
+#include <thread>
+#include <tuple>
+
+#include "MetricsUploader/MockMetricsUploader.h"
+#include "MetricsUploader/SymbolLoadingMetric.h"
+#include "MetricsUploader/orbit_log_event.pb.h"
+
+namespace orbit_metrics_uploader {
+
+using ::testing::_;
+
+namespace {
+
+bool OrbitPerModuleSymbolLoadDataAreEqual(const OrbitPerModuleSymbolLoadData& lhs,
```

Comment:
The error message would be uninformative, not revealing which field has mis-matched. 

I sueegest to either write a custom matcher or just replace this function with `void ExpectOrbitPerModuleSymbolLoadDataEq(lhs, rhs)` that would make three separate calls to `EXPECT_EQ`.